cloud-service-provider/aws/eks/variables.tf

```diff
@@ -0,0 +1,59 @@
+variable "region" {
+  description = "AWS region"
+  type        = string
+  default     = "eu-west-1"
+}
+
+variable "cluster_name" {
+  description = "EKS cluster name"
+  type        = string
+  default     = null
+}
+
+variable "cluster_version" {
+  description = "EKS cluster version"
+  type        = string
+  default     = "1.31"
+}
+
+variable "instance_types" {
+  description = "EC2 instance types"
+  type        = list
+  default     = ["t3.medium"]
+}
+
+variable "use_custom_launch_template" {
+  description = "Disk size in GiB for nodes."
+  type        = bool
+  default     = true
+}
+
+variable "disk_size" {
+  description = "Disk size in GiB for nodes."
+  type        = number
+  default     = 20
```

Comment:
Do we need to add something to readme file based on size of model the disk size needed?

Reply:
That can be added maybe later once it is measured. There are two sizes here. This is about how much we need for the container images. The AWS managed node group default 20 GB is not enough. For chatqna app 50 GB is enough (see `opea-chatqna.tfvars`). The other size is the model size, which is stored in PV/PVC, and is 50 GB hard coded in `eks-efs-csi-pvc.yaml`. Confusingly both numbers happen to be 50.